src/types/payload.ts

```diff
@@ -94,8 +96,18 @@ export interface PlayedTileInteractionPayload {
 }
 
 export interface InteractionSuccessPayload {
+  connectionId?: string;
```

Comment:
did you rebase from master branch? I removed `connectionId` in this payload

Reply:
I did rebase, looks like it added it back in somehow. I've removed it.